tools/frrcommon.sh.in

```diff
@@ -348,11 +354,11 @@ frrcommon_main() {
 		esac
 	else
 		case "$cmd" in
-		start)	daemon_start "$@";;
+		start)	daemon_start "$@" "$@";;
```

Comment:
you are passing all args twice here, I don't think you need to change these line to pass your second argument, it is already taken care of by the first $@